regparser/commands/current_version.py

```diff
@@ -40,6 +42,7 @@ def create_version_entry_if_needed(cfr_title, cfr_part, year):
     version_entries = entry.Version(cfr_title, cfr_part)
 
     if version_id not in version_entries:
+        logger.debug("Writing version %s", version_id)
```

Comment:
We already log when writing to the index: https://github.com/18F/regulations-parser/blob/master/regparser/index/entry.py#L41